src/validation.js

```diff
@@ -109,6 +109,15 @@ export function periodValidation(
 ) {
   const { rnr_list, periods } = outgoingPeriods;
 
+  // allow an unlimited number of emergency orders
+  // and only one normal order per period
+  if (rnr_list && !emergency) {
```

Comment:
With this addition, I think the block from line 128-131 can be removed. Although I am confused as to how this is helping to allow unlimited emergency orders. This only prevents more non-emergency orders than the given max orders per period? The block 128-131 does this also, just after the return of a period for an emergency order?

```
if (rnr_list && rnr_list.length !== 0) {
    const anyRegularOrders = rnr_list.some(rnr => rnr.emergency !== true);
    if (anyRegularOrders) throw errorObject(ERROR_PERIOD_UNFINISHED);
  }
```

Also can we use `numberOfRegularOrders` or something similar in camel case (I should have renamed `rnr_list` when destructuring to be consistent - oops :(

Reply:
hm. I must have been working off a very old version or something. will just abandon - these changes aren't needed at all.